apicast/conf/nginx.conf.liquid

```diff
@@ -0,0 +1,92 @@
+env REDIS_HOST;
```

Comment:
Is `apicast/conf/nginx.conf` still needed?

Reply:
There's a couple of settings present in the original config but not on this template:
```
  lua_max_running_timers 2048;
  lua_socket_pool_size 512;
```

Reply:
aha! they probably lost during the rebase, good catch!

the plain nginx.conf should not be needed

Reply:
Fixed both (removed the old file and added those missing directives).